onward/app/views/fragments/richLinkBody.scala.html

```diff
@@ -91,13 +89,15 @@ <h1 class="rich-link__title">
                 @*<div class="u-h meta__number" data-discussion-id="@content.discussionId"/>*@
             }
         } else { 
-            @content.tags.contributors.headOption.map { profile => 
-                @profile.properties.contributorLargeImagePath.map{ image =>
-                    <div class="rich-link__avatar">
-                        <img class="rich-link__avatar__media" src="@ImgSrc(image, RichLinkContributor)" alt="@profile.name" />
-                    </div>
-                }
-            } 
+            @if(content.content.cardStyle == Comment) {
```

Comment:
Do we ever have contributor images when its not comment?

Reply:
Yes, many times it turns out.

Reply:
Thats what I was thinking. Same comment as above, this changes the current functionality. Are we okay with that?

Reply:
Yep 😃  (only for users in the experiment)